include/sys/zfs_context.h

```diff
@@ -766,4 +766,7 @@ extern void spl_fstrans_unmark(fstrans_cookie_t);
 extern int spl_fstrans_check(void);
 
 #endif /* _KERNEL */
+
+#define	ZFS_ARRAY_SIZE(a)	(sizeof (a) / sizeof (*a))
```

Comment:
@jxiong ARRAY_SIZE is now available in the master branch.  It was added as part of the iostat extensions, so you can drop this from your patch and just use it.